.github/workflows/style.yml

```diff
@@ -0,0 +1,24 @@
+# Run style checks
+
+name: style
+
+on:
+  - pull_request
+  - workflow_dispatch
+
+jobs:
+  style:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          cache: "pip"
+          cache-dependency-path: ".github/workflows/qc-requirements.txt"
+      - run: python -m pip install -r .github/workflows/qc-requirements.txt
+      - run: python -m pip install .
```

Comment:
I think we can get rid of this line completely: this installs the `rounders` package, but we don't need to install it to run style checks. (We don't strictly need to install it for running tests, either, but I think it's good practice, so no need to change the QC workflow.)

```suggestion
```